### ISSUE TITLE
Training Scripts

### ISSUE THREAD

**Issue by opeide** [TRI-ML/realtime_panoptic#12]
Great project! 
Are you going to include training scripts? 
Or perhaps just the loss functions you describe in your paper?

**Comment by jieli352** [comment]
Thank you for your interest! We are working on a training pipeline using detectron2. We will release the code when it is ready.

**Comment by cosminginerica** [comment]
Hello.
Nice work!
Any update on training the network? 

Thank you

**Comment by Zaiwgu** [comment]
Thank you for your excellent work! Any updates on training codes?

**Comment by hiepbk** [comment]
Hello author.
Thank you for your great work.
Can you provide the training and evaluation scripts. 
Thank you
